GraphView/graphview.py

```diff
@@ -707,6 +708,15 @@ def search_config_panel(self, configdialog):
             _("Show bookmarked persons first in search result list."))
 
         return _('Search'), grid
+
+    def font_filter_func(self, family, face):
+        style = face.get_face_name()
+        if style in ('Обычный', 'Regular', 'Normal', 'obyčejné', 'Standard',
+                     'Κανονικά', 'Normaali', 'Normál', 'Normale', 'Standaard',
+                      'Normalny', 'Normálne', 'Navadno', 'thường', 'Arrunta'):
+            return True
+        return False
+
```

Comment:
I'm not sure you should do it this way; we support a lot more languages than this I think.  I suggest:
<pre>
        desc = face.describe()
        stretch = desc.get_stretch()
        if stretch != Pango.Stretch.NORMAL:
            return False  # avoid Condensed or Expanded
        sty = desc.get_style()
        if sty != Pango.Style.NORMAL:
            return False  # avoid italic etc.
        weight = desc.get_weight()
        if weight != Pango.Weight.NORMAL:
            return False  # avoid Bold
        return True
</pre>

Reply:
Yes, your solution is better. Thanks!